stl/inc/format

```diff
@@ -992,7 +995,7 @@ private:
     void _Store(const size_t _Arg_index, const _Ty& _Val) noexcept {
         // clang-format on
         _Store_impl<typename basic_format_arg<_Context>::handle>(
-            _Arg_index, _Basic_format_arg_type::_Custom_type, basic_format_arg<_Context>::handle(_Val));
+            _Arg_index, _Basic_format_arg_type::_Custom_type, typename basic_format_arg<_Context>::handle(_Val));

```

Comment:
Style nitpick: As this is constructing a temporary `handle`, it should use braces to avoid looking like a function call.

Additionally, this statement now mentions `typename basic_format_arg<_Context>::handle` twice. Consider extracting it as a typedef.